### PR TITLE
Updating go version to 1.13

### PR DESCRIPTION
**Pull request by lobkovilya**
Signed-off-by: lobkovilya <ilya.lobkov@xored.com>

## Motivation and Context

`GOPROXY` that we use doesn't have `spiffe/spire` dependency. Go 1.13 has extremely useful feature that allows set `GOPROXY` as comma separated list:
```
GOPROXY=https://proxy.golang.org,direct
``` 
In that case rare dependencies will be downloaded directly.
## How Has This Been Tested?

- [ ] Covered by existing integration testing
- [ ] Added integration testing to cover
- [ ] Tested locally
- [ ] Have not tested

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: controlplane/build/Dockerfile.nsmd [hunk 1]
```diff
@@ -1,8 +1,9 @@
-FROM golang:alpine as build
-
 ARG VERSION=unspecified
 ARG VENDORING
 ARG GOPROXY
+ARG GO_VERSION
+
+FROM golang:${GO_VERSION}-alpine as build
```
Comment (alex-yust) [inline controlplane/build/Dockerfile.nsmd]: How about extracting the whole 'golang:${GO_VERSION}-alpine' thing as an ARG somethinglike BASE_IMAGE?

File: .circleci/config.yml [hunk 2]
```diff
@@ -8,7 +8,7 @@ jobs:
     resource_class: xlarge
     working_directory: /go/src/github.com/networkservicemesh/networkservicemesh
     docker:
-      - image: circleci/golang:1.12-stretch
```
Comment (haiodo) [inline .circleci/config.yml]: Maybe we could use some env variable to put version inside?

Comment (haiodo) [reply .circleci/config.yml]: On other hand, this values here show us a version used, and if will be some env variable in CI we will not know it value.

File: controlplane/build/Dockerfile.nsmd (hunk 1 above)
Comment (haiodo) [reply controlplane/build/Dockerfile.nsmd]: I'm fine with just go version to be passed.

**Pull request merged**